python/lsst/ts/externalscripts/base_take_twilight_flats.py

```diff
@@ -282,6 +295,9 @@ async def configure(self, config: types.SimpleNamespace):
                 if comp in self.camera.components_attr:
                     self.log.debug(f"Ignoring Camera component {comp}.")
                     setattr(self.camera.check, comp, False)
+                elif comp in ["mtdome", "mtdometrajectory"]:
```

Comment:
why make it only `mtdome` and `mtdometrajectory`? shouldn't this be 

```
elif comp in self.tcs.components_attr:
```

?

Reply:
I think the idea here was that we only wanted to allow the option to ignore the mtdome and mtdometrajectory componets. For example, if a user ignored the mtmount, mtdome, and mtdometrajectory, we were worried it would leave open the option for the telescope to be pointed in the direction of the sun while taking data.